rest/blip_client_test.go

```diff
@@ -901,3 +1056,71 @@ func (btc *BlipTesterClient) GetBlipRevMessage(docID, revID string) (msg *blip.M
 
 	return nil, false
 }
+
+func (btc *BlipTesterClient) StartPull() error {
+	return btc.DummyCollection().StartPull()
+}
+
+func (btc *BlipTesterClient) WaitForRev(docID string, revID string) ([]byte, bool) {
+	return btc.DummyCollection().WaitForRev(docID, revID)
+}
+
+func (btc *BlipTesterClient) WaitForBlipRevMessage(docID string, revID string) (*blip.Message, bool) {
+	return btc.DummyCollection().WaitForBlipRevMessage(docID, revID)
+}
+
+func (btc *BlipTesterClient) PullReplication() *BlipTesterReplicator {
+	return btc.pullReplication
+}
+
+func (btc *BlipTesterClient) PushReplication() *BlipTesterReplicator {
+	return btc.pushReplication
+}
+
```

Comment:
I'm not sure these changes were required and would reduce a lot of the diffs in tests if reverted

```suggestion
```

Reply:
Made more sense when the replicators were on the collection clients